build-logic/src/main/kotlin/com/ibm/wala/gradle/JavaCompileUsingEcj.kt

```diff
@@ -65,10 +65,18 @@ abstract class JavaCompileUsingEcj : JavaCompile() {
 
   @TaskAction
   protected override fun compile(inputs: InputChanges) {
+    val testArgs = options.allCompilerArgs
+    val argFile = createTempFile(prefix = "kotlinTemp", suffix = ".tmp")
+    val writer = java.io.PrintWriter(argFile)
```

Comment:
Use Kotlin's `File.printWriter` extension method instead of calling `PrintWriter` constructor directly.